crates/xtask/src/main.rs

```diff
@@ -94,6 +95,18 @@ enum MainCommand {
 
     /// Ensures review can be done in printed form.
     Textreview,
+
+    /// Updates change log of a crate and all dependent crates.
+    Changelog {
+        /// Crate to update
+        crate_path: String,
+
+        // Either "major", "minor", or "patch". TODO(zvv): turn into enum.
+        version: changelog::ChangelogReleaseType,
+
+        /// Message added to the CHANGELOG.md
+        message: String,
+    },
```

Comment:
I'm actually not sure we should start with this feature. I think it would be better to start with the CI check:
- Changelog file is well formatted (as defined in the doc, there are many checks).
- Changelog matches the Cargo.toml (could be done in different PR).

Reply:
Makes sense! sounds good to me, I'll go that direction.

Reply:
I've made updates to create 2 subcommand: `ci` and `change`

* `ci` validates all crates/* that have a changelog.md file
* `change` is unimplementing